lib/sles4sap/sap_deployment_automation_framework/deployment.pm

```diff
@@ -562,8 +563,28 @@ sub prepare_sdaf_project {
     assert_script_run("cd $deployment_dir");
     assert_script_run('mkdir -p ' . log_dir());
 
+    # Calculate SDAF version used for deployment and picks latest -1
+    # SDAF_GIT_AUTOMATION_BRANCH variable will override calculated value
+    my $branch = get_var('SDAF_GIT_AUTOMATION_BRANCH', '');
+    if (!$branch || $branch eq 'latest') {
+        my $tags = script_output("curl -s https://github.com/Azure/sap-automation/tags | grep /releases/tag/ | grep data-view-component | cut -d '/' -f6 | cut -d '\"' -f1");
```

Comment:
You can use directly git command for listing and sorting tags to avoid digging through curl output:
```
git for-each-ref --sort=-version:refname --format '%(refname:short)' refs/tags

v3.13.0.1
v3.13.0.0
v3.12.0.0
v3.11.0.3
v3.11.0.2
v3.11.0.1
v3.11.0.0
v3.10.1.0
v3.10.0.0

```